client/blocks/dismissable-card/README.md

```diff
@@ -0,0 +1,67 @@
+Dismissable Card
+=========
+This is a card component that can be dismissed for a single page load or be hidden
+via user preference.
+
+#### How to use:
+
+```js
+import DismissableCard from 'blocks/dismissable-card';
+import QueryPreferences from 'components/data/query-preferences';
+
+render: function() {
+  return (
+    <div className="your-stuff">
+      <QueryPreferences />
```

Comment:
I'm not sure what we should do about `<QueryPreferences />`. While in the traditional sense, query components should be rendered this way to ensure that the data is kept fresh in response to the dependent component's own lifecycle, we already [render a `<QueryPreferences />` in `<Layout />`](https://github.com/Automattic/wp-calypso/blob/6bb2e62f614bd6d58828b27e8db87a36d4482062/client/layout/index.jsx#L201) so we can be guaranteed that the request will always be made on every page load.